Assets/Scripts/Health.cs

```diff
@@ -4,40 +4,23 @@
 
 public class Health : MonoBehaviour {
 
-	// Use this for initialization
 	public int maxHealth = 10;
+	public int currentHealth;
```

Comment:
keep this private, if you need to display the health in the inspector use the [SerializeField] command to make it visible.

this does not need to be public , especially when you have a getter below